pkg/syncer/metadatasyncer.go

```diff
@@ -1065,7 +1086,21 @@ func csiPVCUpdated(ctx context.Context, pvc *v1.PersistentVolumeClaim,
 	log := logger.GetLogger(ctx)
 	var volumeHandle string
 	var err error
+
+	isMultiVCenterFssEnabled := metadataSyncer.coCommonInterface.IsFSSEnabled(ctx, common.MultiVCenterCSITopology)
```

Comment:
can we have a global variable for FSS check in syncer, similar to multivCenterCSITopologyEnabled in controller pkg ?

Reply:
Created a  global vairable.